plugins/httpinspector.koplugin/main.lua

```diff
@@ -14,6 +9,9 @@ local util = require("util")
 local _ = require("gettext")
 local T = require("ffi/util").template
 
+-- This plugin allows for inspecting KOReader's internal objects,
+-- calling methods, sending events... over HTTP.
```

Comment:
This could still go at the top ftr. ^_^ It's the `---` (or up there, the `--[[--` that marks it as something for the generated docs.

Reply:
That is, `--` or `--[[` is just a comment.